app/controllers/find/feedbacks_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Find
+  class FeedbacksController < ApplicationController
+    def new
+      @feedback_form = FeedbackForm.new
+    end
+
+    def create
+      @feedback_form = FeedbackForm.new(feedback_form_params)
```

Comment:
I think we can remove this form object. We don't need to validate anything in addition to the model validations itself.